packages/sdk-webauthn/index.ts

```diff
@@ -17,14 +17,13 @@ export class WebAuthn implements CredentialSigner<Fido2Assertion>, CredentialSto
       rpId: string
       timeout?: number
     }
-  ) {}
+  ) { }
```

Comment:
is that your linter ?

Reply:
Probably, we should add our standard lint/pre-commit check config